Gravatar/DirectoryImageCache.cs

```diff
@@ -67,16 +67,8 @@ public DirectoryImageCache(string cachePath, int cacheDays)
         {
         }
 
-        /// <summary>
-        /// Occurs whenever the cache is invalidated.
-        /// </summary>
```

Comment:
Could you please clarify what is duplicated here?
Docs for interfaces and implementing classes aren't the same thing.

Reply:
I agree that they don't have to be, but when they are actually the same then they will get out of sync over time.

If implementing members have specific caveats or implementation details to share then they can use `<inheritdoc />` and add further elements. But I consider that a smell because it implies that the implementation cannot be substituted for another.

Here's a simple example of what I shoot for in this kind of pattern.

```c#
/// <summary>Defines a storage system.</summary>
interface IStorage
{
    /// <summary>Removes all items from the store.</summary>
    void Clear();
}

/// <summary>A volatile, in-memory storage system.</summary>
class MemoryStore : IStorage
{
    void IStorage.Clear() { }
}


/// <summary>A persistent storage system backed by the file system.</summary>
class FileSystemStore : IStorage
{
    void IStorage.Clear() { }
}
```

Key point relevant to this discussion is that of the three `Clear` sites, the 'root' gets the documentation. In my IDE at least, `DirectoryImageCache.Invalidated` picks up documentation:

![image](https://user-images.githubusercontent.com/350947/38076387-5f309aa4-332d-11e8-8a2f-9daa3f824fb0.png)


Interested to hear your thoughts.

Reply:
:bulb: Just put `<inheritdoc />` on the implementation methods.

Reply:
It's my understanding that docs are inherited even without `<inheritdoc/>` and that this element is only useful if you want to combine documentation from the base with overriding elements, or if you want to inherit documentation from a different symbol (via `cref` or whatever the attribute is).

Reply:
This might be a R# thing. I've added `<inheritdoc />` to implementations.